package.json

```diff
@@ -39,29 +42,26 @@
     "./component-classes/shortcuts.js",
     "./component-classes/index.d.ts"
   ],
-  "dependencies": {
-    "@warp-ds/tokenizer": "^0.0.4",
-    "@warp-ds/uno": "1.12.0"
-  },
   "devDependencies": {
-    "@eik/cli": "^2.0.33",
-    "@semantic-release/changelog": "^6.0.2",
-    "@semantic-release/exec": "^6.0.3",
-    "@semantic-release/git": "^10.0.1",
-    "@sindresorhus/slugify": "^2.2.1",
-    "@warp-ds/eslint-config": "^1.0.4",
-    "cz-conventional-changelog": "^3.3.0",
-    "eslint": "^8.56.0",
-    "eslint-plugin-prettier": "^5.1.3",
-    "eslint-config-prettier": "^9.1.0",
-    "drnm": "^0.9.0",
-    "lightningcss": "^1.23.0",
-    "prettier": "^3.2.4",
-    "semantic-release": "^23.0.0",
-    "typescript": "^5.3.3",
-    "unocss": "^0.58.5",
-    "vite": "^5.0.12",
-    "vite-plugin-dts": "^3.7.1"
+    "@eik/cli": "2.0.38",
+    "@semantic-release/changelog": "6.0.3",
+    "@semantic-release/exec": "6.0.3",
+    "@semantic-release/git": "10.0.1",
+    "@sindresorhus/slugify": "2.2.1",
+    "@warp-ds/eslint-config": "1.0.5",
+    "@warp-ds/tokenizer": "0.0.4",
+    "@warp-ds/uno": "2.x",
+    "cz-conventional-changelog": "3.3.0",
+    "drnm": "0.9.0",
+    "fs-extra": "11.2.0",
+    "lightningcss": "1.25.1",
+    "prettier": "3.3.2",
+    "semantic-release": "24.0.0",
+    "typescript": "5.5.3",
+    "unocss": "0.x",
```

Comment:
Is it truly safe to assume that only major changes in the `unocss` will break our package? Historically, many minor changes in v0 of `unocss` would introduce some breaking changes. Here's an example: https://github.com/unocss/unocss/releases/tag/v0.59.0

Reply:
Since we have added a peerDependency now:

```
"peerDependencies": {
    "@warp-ds/uno": "2.x"
  },
```

This'll make sure that the version of unocss is same for both these apps

Reply:
I understand but what about the breaking changes that often are introduced as minor changes in that package?

Reply:
We'll have to be a bit vigilant when bumping versions (which we should always do)

Reply:
What do you mean by that? Is it enough to check if build/tests fail? If that's not enough, then what would be a good practice here? I'm just afraid we are making maintaining of Warp packages more error-prone and I'd like to have an idea how to minimise future bugs.